tools/tiara/tiara.xml

```diff
@@ -107,7 +98,7 @@
                 <element name="arc" file="arc" ftype="fasta" />
                 <element name="bac" file="bac" ftype="fasta" />
                 <element name="euk" file="euk" ftype="fasta" />
-                <element name="main_result" file="main_result03.txt" ftype="txt" />
+                <element name="main_result" file="main_result02.txt" ftype="txt" />
```

Comment:
@Minamehr are those files really `txt`? They look to me like a `tabular`

Reply:
This is what in the tool documentation mentioned about the main output: "the probabilities of belonging to individual classes are also written to out.txt". Additionally, in the tool example command, they used "tiara -i sample_input.fasta -o out.txt".  However, as you pointed out, the actual format of the file appears to be tabular with structured rows and columns.